testing/google-cloud-appengine-flex-custom/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM gcr.io/google_appengine/jetty
+ADD google-cloud-appengine-flex-custom-0.11.2-alpha-SNAPSHOT.war $JETTY_BASE/webapps/root.war
```

Comment:
I realize it adds dirt to update this in the version updating script, but having to manually update it is quite disruptive; people who are used to the no-touch commands now would likely forget to update it.